modules/data-entry/src/main/frontend/src/questionnaire/QuestionnaireStyle.jsx

```diff
@@ -449,7 +449,14 @@ const questionnaireStyle = theme => ({
     },
     warningStatus: {
       color: theme.palette.warning.main
-    }
+    },
+    answerOptionInput: {
+      width: "85%"
+    },
+    answerOptionDeleteButton: {
+      right: theme.spacing(-2),
+      float: "right"
```

Comment:
This fixes the broken layout and improves the alignment (the X on the buttons now appears aligned on the right with the inputs above)
```suggestion
      right: theme.spacing(-3),
```